libr/core/cmd_anal.c

```diff
@@ -4562,6 +4575,31 @@ static void cmd_anal_esil(RCore *core, const char *input) {
 			break;
 		}
 		break;
+	case 'l': // ael commands
+		switch (input[1]) {
+		case 'i': // aeli interrupts
+			switch (input[2]) {
+			case ' ': // "aeli" with arguments
+				if (!r_anal_esil_load_interrupts_from_lib (esil, input + 3)) {
+					eprintf ("Failed to load interrupts from '%s'.", input + 3);
+				}
+				break;
+			case 0: // "aeli" with no args
+				if (esil && esil->interrupts) {
+					dict_foreach (esil->interrupts, _aeli_iter, NULL);
```

Comment:
arent we suposed to kill dict_*¿

Reply:
another pr, maybe?
i tried making the most minimal amount of changes